docs/references/making_release.md

```diff
@@ -0,0 +1,18 @@
+How to make a new release on PyPI and conda-forge
+
+PyPI:
+1. iterate the version number
+2. make the distribution: python setup.py sdist
+3. upload to test site: twine upload --repository testpypi dist/*
+4. check that it looks good at https://test.pypi.org/
+5. upload to real site: twine upload --repository pypi dist/*
+
+Conda:
+1. Fork the feedstock repo
```

Comment:
ie fork pyuvdata to your personal github account?

Reply:
yes, I'll add that clarification